jenkins/hru_R_Jenkinsfile

```diff
@@ -12,19 +12,19 @@ pipeline {
     stage('geodatabase simplification') {
       agent {
         docker {
-          image 'code.chs.usgs.gov:5001/wma/iidd/wbeep-data-processing:latest'
+          image 'code.chs.usgs.gov:5001/wma/iidd/wbeep-data-processing:R-latest'
           alwaysPull true
         } 
       }
       steps {
-        sh 'df -h' 
         sh 'Rscript -e "source(\'src/process_hru_shapes.R\')"'
+        sh 'ogr2ogr -f GeoJSON hrus.geojson -lco COORDINATE_PRECISION=5 hru_reduced_valid.shp'
       }
     }
     stage('push to S3') {
       agent any
       steps { 
-        sh 'aws s3 cp topojson_valid_quant.topojson s3://prod-owi-resources/resources/Application/wbeep/development/hru_shape/simple/topojson_valid_quant.topojson'
+        sh 'aws s3 cp hrus.geojson s3://prod-owi-resources/resources/Application/wbeep/development/hru_shape/hrus.geojson --content-type "application/json"'
```

Comment:
FYI the AWS Pipeline plugin is installed so you can use the plugin API 

https://github.com/jenkinsci/pipeline-aws-plugin#s3upload